tests/e2e/env/docker-compose.yml

```diff
@@ -2,8 +2,7 @@ version: '3'
 
 services:
   wordpress:
-    build: ./wordpress-xdebug
-    image: wcp_e2e_wordpress
+    image: wordpress:5.6.2-php7.3
```

Comment:
Could we also update the image to one with PHP v7.4? We specify PHP 7.4 for E2E tests, but the docker images still uses PHP 7.3.

Reply:
This image ID is from the to-be-deleted `Dockerfile`:
https://github.com/Automattic/woocommerce-payments/blob/5d19129845e132624bb08cf8f7bb325803f04516/tests/e2e/env/wordpress-xdebug/Dockerfile#L1

But yeah, I can add it to PHP 7.4. I will update the WP version to `5.8` too as it's our minimum requirement now:
https://github.com/Automattic/woocommerce-payments/blob/5d19129845e132624bb08cf8f7bb325803f04516/woocommerce-payments.php#L13-L14

Reply:
That would be great. Thanks 🤝

Reply:
Addressed here: https://github.com/Automattic/woocommerce-payments/pull/4577/commits/075050509db89848271367782d8b03c868c93468

Reply:
Awesome. Thanks 👍